### PR TITLE
Refactor and add PyTorch models

### PR DESCRIPTION
**Pull request by zphang**
Will rebase once changes are finalized. Should not merge until after rebase.

Changes:

* Added PyTorch as well as CPU options
* Added tests
* Code style changes
* Uploaded PyTorch models

Comments:

* Currently the code is set up to load and run inference on a single set of 4 images. With a little additional work, we can allow the user to supply multiple sets of images, so it can be immediately useful for (batch) inference. Do we want to do this now?
* We're currently uploading the large model files into GitHub. Do we want to think about putting them on Google Drive or S3 instead?

### CODE REVIEW

File: birads_prediction_torch.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import argparse
+import torch
+
+import models_torch as models
+import utils
+
+
+def inference(parameters, verbose=True):
+
```
Comment (seyiqi) [inline birads_prediction_torch.py]: remove space

File: convert_model.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import argparse
+import torch
+import tensorflow as tf
+
+import models_torch
+
+
+def tf_to_torch(input_path, output_path):
```
Comment (seyiqi) [inline convert_model.py:8]: quick comment on this method?

File: convert_model.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import argparse
+import torch
+import tensorflow as tf
+
```
Comment (seyiqi) [inline convert_model.py:4]: remove space

File: convert_model.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Convert from TensorFlow checkpoints to PyTorch pickles')
+    parser.add_argument('input_path')
+    parser.add_argument('output_path')
+    args = parser.parse_args()
+
+    tf_to_torch(args.input_path, args.output_path)
+
+
+"""
```
Comment (seyiqi) [inline convert_model.py]: Is this commented block necessary?

File: layers_torch.py [hunk 5]
```diff
@@ -0,0 +1,91 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class AllViewsGaussianNoise(nn.Module):
```
Comment (seyiqi) [inline layers_torch.py:6]: Give the classes in this file some comments so that it matches layers_tf.py?

File: convert_model.py (hunk 4 above)
Comment (zphang) [reply convert_model.py]: Removing.

File: layers_torch.py (hunk 5 above)
Comment (zphang) [reply layers_torch.py:6]: Will do.

File: convert_model.py (hunk 2 above)
Comment (zphang) [reply convert_model.py:8]: Adding.

File: birads_prediction_torch.py (hunk 1 above)
Comment (zphang) [reply birads_prediction_torch.py]: I think the extra line is required by PEP 8.

**Comment by zphang** [comment]
Pushed a commit.

**Pull request merged**